### DISCUSSION TITLE
Disconnect from live.com

### DISCUSSION THREAD

**Discussion by LongJohnHancock** [community/community#133849] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don't know how I got this account connected to my live.com but it's causing me allot of headaches. How do I unlink my live.com account from my GitHub. Or can a mod just please do it for me. Thank you, John

**Comment by queenofcorgis** [comment]
Hi @LongJohnHancock , we're happy you're here! You are more likely to get a useful response if you are explicit about what your live.com  is, giving a few more details might help someone give you a nudge in the right direction.
